### PR TITLE
Update Profile page

### LINKED ISSUE #28: [Patient] Allow doctor to write the record for patients

### LINKED ISSUE #47: [Profile] Integrate API related to records for Profile page
Integrate the API to Create new record and View record for Profile page

### LINKED ISSUE #51: [Profile] Integrate API to update profile for Profile page
- The profile page will display the information of the user that are viewed
- Integrate the API to update profile from Profile page. Users can only update their own account (which they logged in).

### PR DESCRIPTION
**Pull request by chalalala**
- Display user info
- Allow users to update their own profile
- Display medical records of a specific patient
- Allow doctors to add new record for patients

### CODE REVIEW

**Review by PhuongPhg — COMMENTED** [review]
image
The user information fontsize looks a bit big compare to the fontsize of medical record.  Can you fix that?

File: src/views/registrationform.js [hunk 1]
```diff
@@ -41,7 +72,7 @@ export default function RegistrationForm() {
 							inputVariant="outlined"
 							format="MM/dd/yyyy"
 							id="date-picker-inline"
-							label="Set appointment date"
+							label="Date"
 							value={firstday}
```
Comment (PhuongPhg) [inline src/views/registrationform.js:76]: image
Pls check again the original date format

Comment (PhuongPhg) [reply src/views/registrationform.js:76]: image
And this, too

Comment (chalalala) [reply src/views/registrationform.js:76]: I've changed the format of the date. Please check it.

**Review by PhuongPhg — APPROVED** [review]

**Pull request merged**